chapter-RDD/RDD.Rmd

```diff
@@ -49,149 +60,996 @@ layout: true
+
+rdd_plot <- mlda_long %>%
+  ggplot(aes(x = agecell, y = death_rate, color = over21)) + geom_point() +
+    geom_smooth(mapping = aes(group = over21), se = FALSE, method = "lm",
+              formula = y ~ x) +
+  geom_vline(xintercept = 21, linetype = "longdash") +
+  labs(x = "Age", y = "Death rate (per 100,000)") +
+  scale_colour_viridis_d() +
+  theme_bw(base_size = 16) + theme(legend.position="none") +
+  facet_grid(rows = vars(death_cause), scales = "free_y")
+rdd_plot
+```
+
+---
+
+# Nonparametric Estimation
+
+* Give more weight to observations close to the cutoff level
+
+--
+
+2 settings:
+
+  * How much more weight?
+
+--
+  
+  $\rightarrow$ depends on the chosen ***kernel***.
+
+--
+
+  * How far away from the cutoff do observations need to be to be discarded?
+  
+--
+
+  $\rightarrow$ depends on the chosen ***bandwidth***.
+  
+--
+
+Luckily there's an `R` package that chooses these settings optimally based on fancy algorythms: `rdrobust`.
+
+---
+
+layout: false
+class: title-slide-section-red, middle
+
+# Identifying Assumptions
+
+---
+
+layout: true
+
+<div class="my-footer"><img src="../img/logo/ScPo-shield.png" style="height: 60px;"/></div> 
+
+---
+
+# RDD Assumptions
+
+> *Key assumption*: ***Potential outcomes are smooth at the threshold.***
+
+--
+
+$\rightarrow$ assignment variable cannot be manipulated!
+
+--
+
+Formally:
+
+$$\lim_{r \to c+} E[Y_i^d|r] = \lim_{r \to c-} E[Y_i^d|r], d \in \{0,1\}$$
+
+--
+
+* The population just below must not be discretely different from the population just above the cutoff.
+
+--
+
+* Assumption is violated if people can manipulate the running variable because they know the cutoff value.
+
+--
+
+  * Knowing the cutoff value in itself does not violate the assumption, only ability to manipulate running variable does.
+
+---
+
+# RDD Assumptions
+
+> *Key assumption*: ***Potential outcomes are smooth at the threshold.***
+
+If the assumption holds, we have:
+
+$$
+\begin{align}
+&\lim_{r \to c^+} \mathbb{E}[Y_i | R_i = r] - \lim_{r \to c^-} \mathbb{E}[Y_i | R_i = r] \\
+= &\lim_{r \to c^+} \mathbb{E}[Y_i^1 | R_i = r] - \lim_{r \to c^-} \mathbb{E}[Y_i^0 | R_i = r] \\
+= &\mathbb{E}[Y_i^1 | R_i = c] - \mathbb{E}[Y_i^0 | R_i = c] \\
+= &\mathbb{E}[Y_i^1 - Y_i^0 | R_i = c] \\
+\end{align}
+$$
+
+---
+
+# RDD Assumptions
+
+> *Key assumption*: ***Potential outcomes are smooth at the threshold.***
+
+If the assumption holds, we have:
+
+$$
+\begin{align}
+&\lim_{c \to 21^+} \mathbb{E}[Y_i | a_i = c] - \lim_{a \to 21^-} \mathbb{E}[Y_i | a_i = c] \\
+= &\lim_{c \to 21^+} \mathbb{E}[Y_i^1 | a_i = c] - \lim_{c \to 21^-} \mathbb{E}[Y_i^0 | a_i = c] \\
+= &\mathbb{E}[Y_i^1 | a_i = 21] - \mathbb{E}[Y_i^0 | a_i = 21] \\
+= &\underbrace{\mathbb{E}[Y_i^1 - Y_i^0}_\text{ATE} | a_i = 21] \\
+\end{align}
+$$
+
+---
+
+# Example of Manipulation: French Baccalaureate Grades
+
+```{r, echo = FALSE, out.width = "700px"}
+knitr::include_graphics("../img/photos/manipulation_rdd.png")
```

Comment:
can you explain why that shows manipulation? not immediately clear to me from this. maybe a right column with 2 bullets.

Reply:
This is maybe not the best example of manipulation because it is a bit more intricate but what it shows is that the distribution of average grades at the French baccalaureate is not smooth and that there is bunching around 10, 12, 14 and 16. This is because jurys can decide whether to bump a student who has an average of 15.9 to 16 for example. This decision is probably not completely random and may be correlated with characteristics of individuals that are related to some outcomes of interest. Therefore a simple RDD would yield biased estimates.
Should I change the example to this one: https://pubs.aeaweb.org/doi/pdfplus/10.1257/pol.3.2.41?

Reply:
i didn't think the F example was particularly bad; just needed more explanation. now you made the changes so it's fine.